doc/import_external_docs.ps1

```diff
@@ -2,13 +2,13 @@ Set-PSDebug -Trace 1
 
 $external_docs =
 @(
-    @("https://github.com/unoplatform/uno.wasm.bootstrap", "uno.wasm.bootstrap", "44bb961720b3ae27613b68f7da96ed2a406ef4a8"),
-    @("https://github.com/unoplatform/uno.themes", "uno.themes", "a5ba64eb6967ea2b315adeda77eea452359bb13e"),
-    @("https://github.com/unoplatform/uno.toolkit.ui", "uno.toolkit.ui", "d512cc88a541b2da858e7013b81d8a1b83d89272"),
-    @("https://github.com/unoplatform/uno.check", "uno.check", "e1b260c2e112b7dd92ed77ff46020dd265cd6b90"),
-    @("https://github.com/unoplatform/uno.xamlmerge.task", "uno.xamlmerge.task", "a6d2efa69e24e8280c38300b5c1b7a8f2033f9f9"),
-    @("https://github.com/unoplatform/figma-docs", "figma-docs", "a740582020509f9947fbf991628075a4717bff0a"),
-    @("https://github.com/unoplatform/uno.extensions", "uno.extensions", "fedd58366a66fb30d38c3e9676f9947b055836e6")
+    @("https://github.com/unoplatform/uno.wasm.bootstrap", "uno.wasm.bootstrap", "44bb961720b3ae27613b68f7da96ed2a406ef4a8", "main"),
+    @("https://github.com/unoplatform/uno.themes", "uno.themes", "a5ba64eb6967ea2b315adeda77eea452359bb13e", "master"),
+    @("https://github.com/unoplatform/uno.toolkit.ui", "uno.toolkit.ui", "d512cc88a541b2da858e7013b81d8a1b83d89272", "main"),
+    @("https://github.com/unoplatform/uno.check", "uno.check", "e1b260c2e112b7dd92ed77ff46020dd265cd6b90", "main"),
+    @("https://github.com/unoplatform/uno.xamlmerge.task", "uno.xamlmerge.task", "a6d2efa69e24e8280c38300b5c1b7a8f2033f9f9", "main"),
+    @("https://github.com/unoplatform/figma-docs", "figma-docs", "a740582020509f9947fbf991628075a4717bff0a", "main"),
+    @("https://github.com/unoplatform/uno.extensions", "uno.extensions", "fedd58366a66fb30d38c3e9676f9947b055836e6", "main")
```

Comment:
The branch name will be ignored per the following code (lines 33 to 35 to be more specific):

https://github.com/unoplatform/uno/blob/8965c3093cfd8c5e7ccb9e1369edfce03e508b44/doc/import_external_docs.ps1#L30-L46

If we want to always checkout the default branch, I think we can remove the hash and branch name altogether and remove the `git checkout` step in the script.

But I'm not sure if there is a specific reason to pin the docs to a specific commit.

Reply:
The reason for this change is to alter the branch name for the submodules checked out, instead of the uno repository default branch name. At this point, it's mostly a bug or a missing feature in docfx.